blocktx/store/sql/RegisterTransaction.go

```diff
@@ -95,27 +94,6 @@ func (s *SQL) RegisterTransaction(ctx context.Context, transaction *blocktx_api.
 			return "", nil, 0, err
 		}
 
```

Comment:
This is a bit too rigorous of a change. The idea here was to check whether the transaction had already been mined and return the block hash and height of the block it was mined in.

The discussion we had on this earlier was regarding whether this logic was entirely correct, which it might not have been, but if the transaction has already been mined, that should be returned from this function call, which now has been removed.